src/pages/api/graphql.ts

```diff
@@ -1,31 +1,46 @@
-import { ApolloServer } from "apollo-server-micro";
 import configureCors from "cors";
-import { NextApiRequest, NextApiResponse } from "next";
+import { NextApiHandler, NextApiRequest, NextApiResponse } from "next";
 import { resolvers } from "../../graphql/resolvers";
 import typeDefs from "../../graphql/schema.graphql";
 import { context } from "../../graphql/server-context";
 import { runMiddleware } from "../../lib/run-middleware";
 
-const cors = configureCors();
+import { ApolloServer } from "@apollo/server";
```

Comment:
Are you sure Apollo/Server will persist the cache across requests in NextJS?  My understanding is that its serverless and horizontally scaling so each request starts a new instance.  I think Vercel has a way of caching serverless and edge functions by setting the headers, but there are rate limits and would need to be deployed on vercel 🤷

Reply:
As discussed together, we are not in a serverless context when we deploy :) Thus the cache is persisted across requests.